strr-examiner-web/app/components/ApplicationInfoHeader.vue

```diff
@@ -87,6 +87,10 @@ const nocCountdown = computed(() => {
           <span v-if="!nocCountdown.isExpired">{{ `(${nocCountdown.days} days left)` }}</span>
           <span v-else class="font-bold text-red-500"> (EXPIRED)</span>
         </template>
+        <template v-if="activeHeader.reviewer?.username">
+          | <strong>Assignee:</strong>
```

Comment:
Content should come from a translation file.

Reply:
Done